src/Model/ParameterGo.cs

```diff
@@ -411,7 +419,7 @@ public static string Validate(this IEnumerable<ParameterGo> parameters, HttpMeth
 
                 var name = !p.IsClientProperty
                         ? p.Name.Value
-                        : "client." + p.Name.Value.Capitalize();
+                        : p.GetClientPropertryName();
 

```

Comment:
I presume this is the fix ?

Reply:
This is for generating validation code.  The case that's still causing compile failures is the change on line 55.

Reply:
looks good, thanks!